homework_9/worker.py

```diff
@@ -0,0 +1,54 @@
+# 2. Create a class with a description of the worker. Any worker. employees.
+# I will evaluate the completeness of the described class and will withdraw points for everything unnecessary.
+# I want to see clean code. I expect to see clean code with annotations.
+# So far, no first-class connections with the second.
+# In all methods, I expect to see docstrings with a sane description.
+
+from random import randint
+
+
+class Worker:
+
+    def __init__(self, name: str, surname: str, position: str, salary: float):
+        self.__name = name
+        self.__surname = surname
+        self.__position = position
+        self.__salary = salary
+
+    @property
+    def name(self):
+        return self.__name
+
+    @property
+    def surname(self):
+        return self.__surname
+
+    @property
+    def position(self):
+        return self.__position
```

Comment:
Actually position of worker can be changed/ So it is good to have setter for it

Reply:
I've added the setter on line 30